tdk/set_staff_stores.sql

```diff
@@ -0,0 +1,13 @@
+WITH ranked_stores AS (
```

Comment:
What is this for? Looks very hacky as it works only for the prefixes of `staff` and `stores` having the same size.

Reply:
What happens is that there are circular dependencies in this db and to resolve that cycle TDK sets store_id of staff entity to null. This script "fixes" this empty column. Yes, there is an assumption that tables are the same size. My understanding is that it's actually a business rule - "staff" represents a manager of a store and each store has exactly one manager. So we just do 1-to-1 mapping.